server/package.json

```diff
@@ -1,28 +1,14 @@
 {
-  "name": "@report-bot/server",
+  "name": "server",
   "version": "1.0.0",
-  "description": "Slack bot for scraping the content of messages and storing them in a database.",
-  "main": "src/index.js",
-  "repository": "git@github.com:vacuumlabs/report-bot.git",
-  "author": "Tomas Fabry <tomas.fabry@vacuumlabs.com>",
   "license": "Unlicense",
   "scripts": {
-    "clean": "rimraf dist && rimraf node_modules",
+    "build": "rm -rf dist && babel src -d dist",
```

Comment:
Is there a reason you stopped using `rimraf` everywhere? I think that's a step back.

Reply:
I ran `yarn clean` in the root and it was not working, because `rimraf` was missing in dependencies. So instead of adding new dependency, I used `rm -rf`.
If you think it's worse now, I can add `rimraf` everywhere and use it.

Reply:
I see, leave `rm -rf`. We will add `rimraf` if it is needed.